### PR TITLE
Add missing network icons and node gateway endpoints

### PR DESCRIPTION
**Pull request by BellringerQuinn**

### CODE REVIEW

File: Assets/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "xyz.0xsequence.waas-unity",
-  "version": "2.0.0",
+  "version": "2.1.0",
```
Comment (taylanpince) [inline Assets/package.json:3]: Probably 2.0.1 is better until we add a new feature (i.e. Apple Sign-in is ready)

**Review by taylanpince — CHANGES_REQUESTED** [review]

File: Assets/package.json (hunk 1 above)
Comment (BellringerQuinn) [reply Assets/package.json:3]: I've added a sample, end sessions automatically OnApplicationQuit, and updated supported networks. Shouldn't that be 2.1.0?

Comment (taylanpince) [reply Assets/package.json:3]: Nope, assuming we are following semver, unless we add net new features should be smallest number increase.

Comment (BellringerQuinn) [reply Assets/package.json:3]: Ok my mistake

**Review by taylanpince — APPROVED** [review]

**Pull request merged**